source/dub/version_.d

```diff
@@ -1,2 +1,2 @@
 module dub.version_;
-enum dubVersion = "v1.12.0";
```

Comment:
Don't commit this

Reply:
I was just about to ask about that.

Reply:
I don't know how that got changed. I fixed it now. Thanks!

Reply:
Probably `git commit -am` ;)
This is updated during the build.